sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -0,0 +1,3270 @@
+        "operationId": "JobClient_QueryJobs",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "jobType",
+            "in": "query",
+            "description": "Job Type.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "jobStatus",
+            "in": "query",
+            "description": "Job Status.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the Query Result object",
+            "schema": {
+              "$ref": "#/definitions/QueryResult"
+            }
+          }
+        },
+        "summary": "Query an IoT hub to retrieve information regarding jobs using the IoT Hub query language"
+      }
+    },
+    "/devices/{id}/modules": {
+      "get": {
+        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "operationId": "RegistryManager_GetModulesOnDevice",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Device ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the Device object",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Module"
+              }
+            }
+          }
+        },
+        "summary": "Retrieve all the module identities on the device."
+      }
+    },
+    "/devices/{id}/modules/{mid}": {
```

Comment:
Did we decide asking them to change the parameter name was non-breaking? From mid to moduleId?

Reply:
We haven't considered asking about that, no